src/main/java/seedu/address/logic/commands/AssignmentCommand.java

```diff
@@ -24,9 +24,8 @@ public class AssignmentCommand extends Command {
     public static final String MESSAGE_FAIL = "Assignment not created.";
     public static final String MESSAGE_USAGE = COMMAND_WORD
         + ": Creates an assignment for all students.\n"
-        + "Parameters: "
-        + "[" + PREFIX_NAME + "ASSIGNMENT_NAME] "
-        + "[" + PREFIX_MAX_SCORE + "MAX_SCORE] "
+        + "Compulsory Parameters: "
+        + PREFIX_NAME + "ASSIGNMENT_NAME " + PREFIX_MAX_SCORE + "MAX_SCORE "
```

Comment:
Nice to see that the docs adhere to the user guide of optional vs compulsory parameters.